tests/test_table_pixel_size_widget.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, cast
+
+from qtpy.QtWidgets import QLineEdit
+
+if TYPE_CHECKING:
+    from pytestqt.qtbot import QtBot
+
+
+OBJECTIVE_LABEL = 0
+RESOLUTION_ID = 1
+CAMERA_PX_SIZE = 2
+MAGNIFICATION = 3
+IMAGE_PX_SIZE = 4
+ROW = 0  # "Nikon 40X Plan Fluor ELWD"
+
+
+def _get_wdg(table):
+    obj = table.item(ROW, OBJECTIVE_LABEL).text()
+    resID = cast(QLineEdit, table.cellWidget(ROW, RESOLUTION_ID))
+    mag = cast(QLineEdit, table.cellWidget(ROW, MAGNIFICATION))
+    cam_px = cast(QLineEdit, table.cellWidget(ROW, CAMERA_PX_SIZE))
+    img_px = cast(QLineEdit, table.cellWidget(ROW, IMAGE_PX_SIZE))
+    return obj, resID, mag, cam_px, img_px
```

Comment:
i'm curious... why did you do these casts?  because you're simply returning them, and because `_get_wdg` itself has no return annotations, they would appear to be doing nothing?